src/clojure_lsp/main.clj

```diff
@@ -430,6 +431,24 @@
         (end
           (apply #'handlers/extension method args)))))
 
+(defn process-alive?
+  [pid]
+  (let [{:keys [exit]} (shell/sh "kill" "-0" (str pid))]
```

Comment:
Should we wrap the call with a try-catch block? Not sure if `shell/sh` could throw an exception

Reply:
I'm going to check and add a try/catch if necessary.